Client/Frontend/Fakespot/Views/HighlightsCardView.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+struct HighlightsCardViewModel {
+    let cardA11yId: String = AccessibilityIdentifiers.Shopping.HighlightsCard.card
+    let footerTitle: String
+    let footerActionTitle: String
+    let footerA11yTitleIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerTitle
+    let footerA11yActionIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerAction
+
+    var footerModel: ActionFooterViewModel {
+        return ActionFooterViewModel(title: footerTitle,
+                                     actionTitle: footerActionTitle,
+                                     a11yTitleIdentifier: footerA11yTitleIdentifier,
+                                     a11yActionIdentifier: footerA11yActionIdentifier)
+    }
+}
+
+class HighlightsCardView: UIView, ThemeApplicable {
+    private struct UX {
+        static let cardBottomSpace: CGFloat = 16
+        static let footerHorizontalSpace: CGFloat = 16
+    }
+
+    private lazy var cardContainer: ShadowCardView = .build()
+    private lazy var contentView: UIView = .build()
+    private lazy var footerView: ActionFooterView = .build()
+
+    private var viewModel: HighlightsCardViewModel?
```

Comment:
Why are we keeping the viewModel around? Is it used after the configure model is called?

Reply:
Also technically this is not a view model, it's a configuration class that hold values and no actions to react to.

Reply:
You're right - no need to store it in this case. I remove it.